backend/sozisel/lib/sozisel/model/polls/poll.ex

```diff
@@ -25,15 +30,74 @@ defmodule Sozisel.Model.Polls.Poll do
 
   @type t :: %__MODULE__{
           question: String.t(),
-          options: [PollOption.t()]
+          options: [PollOption.t()],
+          is_multi_choice: boolean()
         }
 
   @primary_key false
   embedded_schema do
     field :question, :string
+    field :is_multi_choice, :boolean
     embeds_many :options, PollOption, on_replace: :delete
   end
 
+  def changeset(schema, attrs) do
+    schema
+    |> cast(attrs, [:question, :is_multi_choice])
+    |> validate_required([:question, :is_multi_choice])
```

Comment:
Maybe a better solution would be not to require this value, but in the absence you can assign the default value